packages/p2p/src/utils/chat-message.js

```diff
@@ -36,5 +50,9 @@ export const convertFromChannelMessage = channel_message => {
         name: channel_message.name,
         sender_user_id: channel_message.sender.userId,
         url: channel_message.url,
+        custom_type: channel_message.customType,
     });
 };
+
+export const admin_message =
+    "Hello! This is where you can chat with the counterparty to confirm the order details.\nNote: In case of a dispute, we'll use this chat as a reference.";
```

Comment:
did we also remove the message in the footer?

Reply:
updated.